### DISCUSSION TITLE
flashing the classification demo still doesn't work

### DISCUSSION THREAD

**Discussion by noahzn** [bitcraze/discussions#556] [Q&A] [answered]
I updated the firmware to 2022.12, and tried to flash the bootloader again (because my AIdeck is Rev.C). Then I used cfloader to flash the model. The crazyflie rebooted, and...

`Reset to bootloader mode...` 
`Could not save cache, no writable directory  `
`Could not save cache, no writable directory  `
`Skipping bcAI:esp, not in the target list  `
`Deck bcAI:gap8, reset to bootloader  `
`| 0% Writing to bcAI:gap8 deck memory  `

It's always 0%.. please help me. Thanks!

**Comment by hmllr** [comment, marked as answer]
Hi :)
Does your question #557 mean this is solved? If yes, can you please update here with your solution? If no, I would suggest to update all firmwares again before trying to update the gap8. The STM32, nRF and ESP32  all have to be up to date for this to work. You can find a guide how to do this here: https://www.bitcraze.io/documentation/tutorials/getting-started-with-aideck/
Hope this helps,
Hanna

**Reply by noahzn** [reply]
Hi, yes! :) I tried about ten times, and it worked once time. I just restarted many times and unplugged the radio several times.. Thank you.

**Reply by hmllr** [reply]
Good to hear that! Maybe still try to update, it shouldn't take ten tries (there used to be an issue that you needed to wait around 30'' after turning it off for all capacitors to discharge, which led to such behavior, but this is now automatically done in the update, so shouldn't happen anymore).
